screens/YourAnnouncements.screen.jsx

```diff
@@ -0,0 +1,81 @@
+import {
+  Dimensions, FlatList, Platform, View, StyleSheet,
+} from 'react-native';
+import { FAnnouncementCard } from 'components/Scoped/Announcement/FAnnouncementCard';
+import React, { useEffect, useState } from 'react';
+import { useErrorModal } from 'hooks/useErrorModal';
+import sizes from 'themes/sizes';
+import colors from 'themes/colors';
+import { FSpinner } from 'components/Composition/FSpinner';
+import { getMyAnnouncementsService } from '../services/announcement/getMyAnnouncements.service';
+
+export const YourAnnouncementsScreen = () => {
```

Comment:
Ogólnie wygląda okey, więc te listy zamiast za kazdym razem robić wszystko tak samo, to wywal te listy jako jeden komponent np AnnouncementList i daj mu takie propsy żeby z łatwością można było rozróznic co kiedy ma sie dziać czyli np props isMe jako bool. I teraz jeżeli jest na true to robisz takiego fetcha jak jest a jak nie to mu zmieniasz na pobieranie ogłoszeń innego użytkownika. Tak samo bedzie sie wyświetlał komunika jeżeli nie ma żadnych innych ogłoszeń.


Pasowałony też dac jakiś tytuł w headerze w nawigacji na jakiej liście jesteś czyli np jeżeli nie na swojej to Ogłoszenia użytwkonika Dupa a jak na moim to np Moje ogłoszenia. Na  razie sprawdź jak to wygląda a póxbniej podeślij i zobaczymy czy jest sens tak robić. Jak nie to trzeba bedzie to dac jako sticky headeing w flat liscie :D

Reply:
Bo tam jest już przycisk powrót wiec to może mega słabo wyglądać wiec od razu na Twoim miejscu bym zrobiła to sticky header w liście. Wzoruj się na FmultiSelectOptions, tam jest sticky inpput